mbu/module1/src/main/java/com/epam/jmp/fileshare/dao/impl/DiskFileShareDaoImpl.java

```diff
@@ -0,0 +1,180 @@
+
+    private static final String NAME_PROPERTY_POSTFIX = ".name";
+    private static final String EXTENSION_PROPERTY_POSTFIX = ".extension";
+    private static final String EXPIRATION_PROPERTY_POSTFIX = ".expiration";
+
+    private static final String PROPERTY_FILE_EXTENSION = ".properties";
+    private static final String REPOSITORY_LOCATION = "repositoryLocation";
+    private static final String REPOSITORY_NAME = "repositoryName";
+    private static final String DEFAULT_FILE_EXPIRATION_PERIOD = "defaultFileExpirationPeriod";
+    @Autowired
+    @Qualifier("disk-dao.properties")
+    private Properties defaultRepositoryProperties;
+    private Properties repositoryProperties;
+
+    private String repositoryLocation;
+    private String repositoryName;
+    private int defaultFileExpirationPeriod;
+
+    @PostConstruct
+    private void init() throws FileShareException {
+        if (this.defaultRepositoryProperties == null || CollectionUtils.isEmpty(this.defaultRepositoryProperties.values())) {
+            throw new FileShareException("Repository Properties not exists");
+        }
+
+        this.repositoryLocation = this.defaultRepositoryProperties.getProperty(REPOSITORY_LOCATION);
+        this.repositoryName = this.defaultRepositoryProperties.getProperty(REPOSITORY_NAME);
+        this.defaultFileExpirationPeriod = Integer.valueOf(this.defaultRepositoryProperties.getProperty(DEFAULT_FILE_EXPIRATION_PERIOD));
+
+        final File repository = new File(getRepositoryPath());
+        if (!repository.exists()) {
+            repository.mkdirs();
+        }
+
+        final String repositoryPropertiesPath = getRepositoryPropertiesPath();
+        final File repositoryPropertiesFile = new File(repositoryPropertiesPath);
+        if (!repositoryPropertiesFile.exists()) {
+            try {
+                repositoryPropertiesFile.createNewFile();
+            } catch (final IOException ex) {
+                throw new FileShareException("Failed to create repository property file", ex);
+            }
+        }
+
+        FileInputStream fis = null;
+        try {
+            fis = new FileInputStream(repositoryPropertiesPath);
+            this.repositoryProperties = new Properties();
+            this.repositoryProperties.load(fis);
+        } catch (final Exception ex) {
+            throw new FileShareException("Failed to load repository properties", ex);
+        } finally {
+            IOUtils.closeQuietly(fis);
+        }
+    }
+
+    @Override
+    public List<FileDto> loadAllFiles() throws FileShareException {
+
+        final File repositoryFolder = new File(getRepositoryPath());
+        final String[] repositoryUuids = repositoryFolder.list(new FilenameFilter() {
+            @Override
+            public boolean accept(final File dir, final String name) {
+
+                return isUUID(name);
+            }
+
+            private boolean isUUID(final String name) {
```

Comment:
In file repository all uploaded files have UUID name without extension.
Method `isUUID` recognizes is file have valid name or not